src/store/audioPlayer.ts

```diff
@@ -0,0 +1,168 @@
+import { createPartialStore } from "./vuex";
+import { AudioPlayerStoreState, AudioPlayerStoreTypes } from "./type";
+import { AudioKey } from "@/type/preload";
+
+export const audioPlayerStoreState: AudioPlayerStoreState = {
+  nowPlayingAudioKeys: [],
+  nowPlayingContinuouslyAudioKey: undefined,
+};
+
+// AudioKey 毎に一つ
+const audioElements: Map<AudioKey, HTMLAudioElement> = new Map();
+
+export const audioPlayerStore = createPartialStore<AudioPlayerStoreTypes>({
+  AUDIO_CURRENT_TIME: {
+    getter: () => (audioKey: AudioKey) => {
+      const audioElement = audioElements.get(audioKey);
+      if (audioElement === undefined)
+        throw new Error("audioElement === undefined");
+      return audioElement.currentTime;
+    },
+  },
+
+  IS_PLAYING_CONTINUOUSLY: {
+    getter(state) {
+      return state.nowPlayingContinuouslyAudioKey !== undefined;
+    },
+  },
+
+  LOAD_AUDIO_PLAYER: {
+    action(_, { audioKey, blob }: { audioKey: AudioKey; blob: Blob }) {
+      let audioElement = audioElements.get(audioKey);
+      if (audioElement === undefined) {
+        audioElement = new Audio();
+      }
+      audioElement.pause();
+      audioElement.src = URL.createObjectURL(blob);
+      audioElements.set(audioKey, audioElement);
+    },
+  },
```

Comment:
この関数は3つのことをやってますね！
1つが`audioElement`の作成、1つが`blob`の代入、もう1つが`audioElements`への登録。
それが`LOAD`という名称から予測できないというのがちょっとだけ気になりました！

全部まとめて`PREPARE_`と呼んじゃうのとかどうでしょう。（最高に良い名前だとは思ってないので他にあれば。。。）
ただ結局その3つを司っていることは分かりづらいので、ドキュメントを書いてあげるといいのかなと思いました！
こんな感じ？
```suggestion
  /**
   * Audioインスタンスを作成して音声blogをセットし、`audioElements`へ登録する。
   */
  PREPARE_AUDIO_PLAYER: {
    action(_, { audioKey, blob }: { audioKey: AudioKey; blob: Blob }) {
      let audioElement = audioElements.get(audioKey);
      if (audioElement === undefined) {
        audioElement = new Audio();
      }
      audioElement.pause();
      audioElement.src = URL.createObjectURL(blob);
      audioElements.set(audioKey, audioElement);
    },
  },
```